src/main/java/uk/gov/pay/api/app/PublicApi.java

```diff
@@ -102,6 +102,8 @@ public void run(PublicApiConfig configuration, Environment environment) {
         environment.jersey().register(injector.getInstance(TransactionsResource.class));
         environment.jersey().register(injector.getInstance(TelephonePaymentNotificationResource.class));
         environment.jersey().register(new InjectingValidationFeature(injector));
+        
+        environment.jersey().register(environment.getValidator());
```

Comment:
Is it required? What for?

Reply:
It's not required apologies I forgot to remove this - it's now done.